packages/components/src/Tile/subcomponents/GenericTile/GenericTile.tsx

```diff
@@ -80,7 +94,7 @@ export const GenericTile = ({
       {information && (
         <div className={styles.informationBtn}>
           <IconButton
-            label="Information"
+            label={`${translatedInfoLabel} ${title}`}
```

Comment:
I just thought of one problem that we'll come into here. If a user does provide a custom label, we're going to be passing in the Tiles `title` regardless 🤔 That could mean they do something like:

```tsx
<InformationTile 
  {...otherProps}
  title="Performance Insights"
  infoButtonLabel="Reveal more information for Performance Insights"
/>
```

That would create an accessible label saying:

"Reveal more information for Performance Insights Performance insights"

Reply:
Made a small change

Reply:
Made a change, what do you think?